crypto/store/loader_file.c

```diff
@@ -179,6 +179,7 @@ static OSSL_STORE_INFO *try_decode_PKCS12(const char *pem_name,
     OSSL_STORE_INFO *store_info = NULL;
     STACK_OF(OSSL_STORE_INFO) *ctx = *pctx;
 
+    *matchcount = 0;
```

Comment:
This isn't strictly necessary, as the calling functions (`file_load_try_decode` and `file_load_try_repeat`) initialise this number to zero before calling these try_decode methods.